FlowCrypt/App/AppContext.swift

```diff
@@ -131,3 +140,35 @@ class AppContext {
         )
     }
 }
+
+class UserContext: AppContext {
```

Comment:
This is not what I meant, but it's not bad. Why not call it `AppContextWithUser`. Then inside controllers, you put `appContext: AppContextWithUser` to make the intention clear. I think it's a good compromise - the variable will be called `appContext` everywhere for consitency, and the type `AppContext` vs `AppContextWithUser` will define if there's user or not. So far so good, to my eye.

Reply:
I like this idea of `AppContext` subclass for authorised users 👍

So just need to rename it to `AppContextWithUser` and `userContext` variable to `appContext` in view controllers as proposed by @tomholub. Otherwise it looks good to me too.